client/src/pages/Login.js

```diff
@@ -1,20 +1,94 @@
-import React from 'react';
-import LoginComponent from '../components/Login/index'
+import axios from 'axios';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { LOADING, SET_USER } from '../store/actions';
+import { useStoreContext } from '../store/store';
 import { Container, Row, Col, Form, Button } from 'react-bootstrap';
```

Comment:
from react-bootstrap's documentation for [importing components](https://react-bootstrap.github.io/getting-started/introduction#importing-components):

> You should import individual components like: react-bootstrap/Button rather than the entire library. Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client.